File: frontend/test_upload.js
async function testUpload() {
    const formData = new FormData();
    const blob = new Blob(["hello world"], { type: "text/plain" });
    formData.append("file", blob, "test.txt");
    formData.append("folderId", "1");
    formData.append("userId", "0");

    try {
        const response = await fetch("http://localhost:8080/api/filesystem/files/add", {
            method: "POST",
            body: formData,
        });
        const text = await response.text();
        console.log("Status:", response.status);
        console.log("Body:", text);
    } catch (e) {
        console.error(e);
    }
}
testUpload();

File: backend/test_upload.py
import requests

url = "http://localhost:8080/api/filesystem/files/add"
files = {'file': ('test.txt', 'hello world', 'text/plain')}
data = {'folderId': 1, 'userId': 0}

try:
    response = requests.post(url, files=files, data=data)
    print("Status Code:", response.status_code)
    print("Response JSON:", response.text)
except Exception as e:
    print(e)
